examples/gpbp_showcase.ipynb

```diff
@@ -27,29 +27,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Retrieving data for Timor-Leste of granularity level 0\n",
-      "Extracting geometry for Timor-Leste\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "adm_area = AdmArea(country=\"Timor-Leste\", level=0)"
+    "adm_area = AdmArea(country=\"Timor-Leste\", level=1)"
```

Comment:
Since we're looking at a smaller administrative area, this needs to be changed to level 1.